patches/api/0312-Missing-Entity-Behavior-API.patch

```diff
@@ -69,6 +69,39 @@ index 0d88dce9978243a1f995c5fb448c5d71b01136eb..cad47139de57642fb3bb483e7a5acaa7
 +    public void setEating(boolean eating);
 +    // Paper end - Horse API
  }
+diff --git a/src/main/java/org/bukkit/entity/Bee.java b/src/main/java/org/bukkit/entity/Bee.java
+index adb20a9abba33c32d553f620fa82b27dff64ab5f..ca6baec5ce00b4d169ab4ff416f616db32615010 100644
+--- a/src/main/java/org/bukkit/entity/Bee.java
++++ b/src/main/java/org/bukkit/entity/Bee.java
+@@ -93,4 +93,28 @@ public interface Bee extends Animals {
+      * @param ticks Ticks the bee cannot enter a hive for
+      */
+     void setCannotEnterHiveTicks(int ticks);
++
++    // Paper start
++    /**
++     * Sets the override for if the bee is currently rolling.
++     *
++     * @param rolling is rolling, or unset for vanilla behavior
++     */
++    void setRollingOverride(@org.jetbrains.annotations.NotNull net.kyori.adventure.util.TriState rolling);
```

Comment:
would be cool to have a line here for what exactly its overriding, this is a bit confusing as is